### PR TITLE
253 labels information on boundingboxes3d

### PR DESCRIPTION
**Pull request by Ardorax**
Write Labels next to BoundingBoxes3D.

* ***Labels on  BoundingBoxes3D*** : Display on render.
```python
import numpy as np
import torch
from aloscene import Frame, Labels, BoundingBoxes3D, CameraExtrinsic, CameraIntrinsic, BoundingBoxes2D

frame = Frame(np.zeros((3, 500, 500)), normalization="01")
frame.append_cam_intrinsic(CameraIntrinsic(focal_length=721.0, plane_size=frame.HW))
frame.append_cam_extrinsic(CameraExtrinsic(torch.eye(4, dtype=torch.float32)))
label = Labels([1, 0, 2], labels_names=["car", "person", "truck"])
label2 = Labels([1, 1, 0], labels_names=["register", "unknown"])

box3d = BoundingBoxes3D([[15, 30, 400, 20, 16, 18, 1], [0, 0, 200, 80, 46, 18, 0.5], [10, 10, 200, 80, 46, 18, 0.5]])
box3d.append_labels(label, "objects")
box3d.append_labels(label2, "type")
frame.append_boxes3d(box3d)

box2d = BoundingBoxes2D(
    [[300, 350, 50, 50], [300, 300, 200, 200], [350, 350, 200, 200]], boxes_format="xcyc", absolute=True, frame_size=(500, 500)
)
box2d.append_labels(label, "objects")
box2d.append_labels(label2, "type")
frame.append_boxes2d(box2d)

frame.get_view().render()
```

* ***Labels dont render*** : #253 

_____
This pull request includes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: aloscene/bounding_boxes_3d.py [hunk 1]
```diff
@@ -500,7 +500,30 @@ def get_view(self, frame, size: Union[tuple, None] = None, mode: str = "3D", **k
...
+                    for label in labels_list:
+                        text.append(
+                            label.labels_names[int(label[index])] if label.labels_names else str(int(label[index]))
+                        )
+                    class_names.append(" , ".join(text))
+                    labels_sum = sum([int(label[index]) for label in labels_list])
+                    color.append(self._GLOBAL_COLOR_SET[labels_sum % len(self._GLOBAL_COLOR_SET)])
+                labels = range(len(class_names))
+            else:
+                labels = [int(label_value) for label_value in self.labels]
```
Comment (Data-Iab) [inline aloscene/bounding_boxes_3d.py:521]: is this case you need to put back the values of labels to the good range, otherwise an out of index error is raised. You can reproduce this error by running your code [here](https://github.com/Visual-Behavior/aloception-oss/issues/253)

**Review by Data-Iab — CHANGES_REQUESTED** [review]
Thank you for your last PR here, hope it's not gonna be your last
